fastmri/pl_modules/data_module.py

```diff
@@ -201,14 +201,15 @@ def _create_data_loader(
             if is_train:
                 sampler = torch.utils.data.DistributedSampler(dataset)
             else:
-                sampler = fastmri.data.VolumeSampler(dataset)
+                sampler = fastmri.data.VolumeSampler(dataset, shuffle=False)
 
         dataloader = torch.utils.data.DataLoader(
             dataset=dataset,
             batch_size=self.batch_size,
             num_workers=self.num_workers,
             worker_init_fn=worker_init_fn,
             sampler=sampler,
+            shuffle=is_train,
```

Comment:
Please alter this to only set `shuffle` if we're not using a sampler. See [here](https://pytorch.org/docs/stable/data.html#torch.utils.data.DataLoader) for details.

Reply:
Yup agreed, I just ran into the bug actually:

```python
ValueError: sampler option is mutually exclusive with shuffle
```

Reply:
corrected